odo/backends/dask.py

```diff
@@ -53,10 +53,18 @@ def resize(x, size):
         return x.resize(size)
 
 
-@convert.register(Array, tuple(arrays), cost=1.)
+@convert.register(Array, tuple(arrays), cost=3.0)
 def array_to_dask(x, name=None, chunks=None, **kwargs):
     if chunks is None:
-        raise ValueError("chunks cannot be None")
+        # chunk into chunks of 1,000 elements along each axis.
+        cs = 1e3
```

Comment:
this seems a little arbitrary, what's wrong with enforcing that the user provides chunks? not providing a default chunks if we're going through dask unintentionally is also nice because then we get the error message